assets/config/funcs.php

```diff
@@ -98,4 +98,31 @@ function getJobNames($unique) {
 	}
 	return $jobNames;
 }
+
+function hashPassword($password, $algorithm, $salt) {
+	switch (strtolower($algorithm)) {
```

Comment:
why is `strtolower` needed?

Reply:
This is just me being defensive before doing the string comparison below. Practical use case; the user inserts into `hash_algorithm` directly, in which case we can't be sure what they've done. So the best we can do is forgive case sensitivity. 

Technically if they just use the website correctly this should never happen. I'm happy to remove it if you like, although I think it's good practice.